workbench-core/accounts/src/utilities/projectUtils.test.ts

```diff
@@ -0,0 +1,526 @@
+/*
+ *  Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *  SPDX-License-Identifier: Apache-2.0
+ */
+
+import { ProjectStatus } from '../constants/projectStatus';
+import { ProjectFilter, ProjectSort } from '../models/listProjectsRequest';
+import Project from '../models/project';
+import { manualFilterProjects, manualSortProjects } from './projectUtils';
+
+describe('projectUtils', () => {
+  let projects: Project[];
+
+  beforeEach(() => {
+    projects = [
+      {
+        id: '',
+        name: 'name1',
+        description: '',
+        costCenterId: 'cc-1',
+        status: ProjectStatus.AVAILABLE,
+        createdAt: '2022-11-10T04:19:00.000Z',
+        updatedAt: '',
+        awsAccountId: '',
+        envMgmtRoleArn: '',
+        hostingAccountHandlerRoleArn: '',
+        vpcId: '',
+        subnetId: '',
+        environmentInstanceFiles: '',
+        encryptionKeyArn: '',
+        externalId: '',
+        accountId: ''
+      },
+      {
+        id: '',
+        name: 'name2',
+        description: '',
+        costCenterId: 'cc-2',
+        status: ProjectStatus.SUSPENDED,
+        createdAt: '2022-11-10T04:20:00.000Z',
+        updatedAt: '',
+        awsAccountId: '',
+        envMgmtRoleArn: '',
+        hostingAccountHandlerRoleArn: '',
+        vpcId: '',
+        subnetId: '',
+        environmentInstanceFiles: '',
+        encryptionKeyArn: '',
+        externalId: '',
+        accountId: ''
+      },
+      {
+        id: '',
+        name: 'name3',
+        description: '',
+        costCenterId: 'cc-3',
+        status: ProjectStatus.DELETED,
+        createdAt: '2022-11-10T04:21:00.000Z',
+        updatedAt: '',
+        awsAccountId: '',
+        envMgmtRoleArn: '',
+        hostingAccountHandlerRoleArn: '',
+        vpcId: '',
+        subnetId: '',
+        environmentInstanceFiles: '',
+        encryptionKeyArn: '',
+        externalId: '',
+        accountId: ''
+      }
+    ];
+  });
+
+  describe('manualSortProjects', () => {
+    let sortRequest: ProjectSort;
+    let expectedProjects: Project[];
+
+    test('should return original list if no sort passed in', async () => {
+      expect(manualSortProjects(sortRequest, projects)).toEqual(projects);
```

Comment:
Yup, as we have discussed in other PRs to this branch, I would like your help refactoring these tests before the feature branch is merged to develop. I am focusing on moving fast right now, and this is how I know how to write test cases. I definitely want your insight/pairing if you're open to it to help me with the refactor because I haven't written really DRY unit tests before. I have been trying to reduce duplication where I can (even these tests are better than past suites IMO), but I need a little more guidance and wanted to tackle them all at once.